templates/src/command.ts.ejs

```diff
@@ -36,7 +36,7 @@ hello world from ./src/<%- name %>.ts!
     const name = flags.name || 'world'
     this.log(`hello ${name} from <%- path.replace(/\\/g, '\\\\') %>`)
     if (args.file && flags.force) {
-      this.log(`you input --force and --file: ${args.file}`)
+      this.log(`you input --force and a file: ${args.file}`)
```

Comment:
I still think we should make `file` a more interesting argument. maybe the greeting word instead of "hello" or something like that.